sagemaker-remote-function/quick_start/quick_start.ipynb

```diff
@@ -0,0 +1,265 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "# Introduction\n",
+    "\n",
+    "We're introducing a new capability in AWS SageMaker Python SDK that allows data scientists to run their Python functions as SageMaker Jobs to take advantage of the compute power offered by SageMaker. This sample notebook is a quick introduction to this new capability with dummy Python functions.\n",
+    "\n",
+    "Before running the samples here, please follow the quick start guide to set up the environment properly, including downloading and installing the private version of SageMaker Python SDK.\n"
```

Comment:
I don't think this line applies any more
`please follow the quick start guide to set up`

There is no quick start guide , this should have link of the document that explains all the setup needed for pathways